src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Text from './Text';
+
+export default {
+  title: 'Text',
+  component: Text,
+};
+
+const bgStyle: React.CSSProperties = {
+	backgroundColor: '#e4e4e4',
+	display: 'inline-block', // Shrink wrap
+	padding: '4px 0'
+};
+
+const rowStyle: React.CSSProperties = {
+	padding: '4px 8px'
+}
+
```

Comment:
Maybe we can move the styles into _Text.scss file?

Reply:
These styles aren't part of the component, they're only for making it look nice in Storybook.  You can see something similar done in `Button.stories.tsx`, there's just a little more styling here due to the need for a background to show the components on.

Reply:
Got it, thanks for the clarification.